GameOfLife/Bipeds.py

```diff
@@ -0,0 +1,68 @@
+from pprint import pprint
+
+
+class Human:
+    def __init__(self, name, birthday, gender):
+        self.name = name
+        self.birthday = birthday
+        self.gender = gender
+
+
+class Population:
+    def __init__(self, humans):
+        self.pop_list = humans
+
+    def filter_gender(self, value):
+        fgend_list = []
+        value = value.lower()
+        for people in self.pop_list:
+            if people.gender.lower() == value:
+                fgend_list.append(people)
+        fgend_list.sort()

```

Comment:
Humans shouldn't be sortable as we haven't implemented anything on Humans to determine the order of a human. Strings and int and floats all have methods that helps finding which one goes first or not. We haven't done the same to Human. It's only sorting pointers if it's even sorting anything at all.